core-bundle/tests/DependencyInjection/Compiler/AddAssetsPackagesPassTest.php

```diff
@@ -126,7 +127,7 @@ public function testUsesTheDefaultVersionStrategyForBundles(): void
 
     public function testUsesTheJsonManifestVersionStrategyForBundles(): void
     {
-        $bundlePath = static::getTempDir().'/ManifestJsonBundle';
+        $bundlePath = Path::normalize(static::getTempDir()).'/ManifestJsonBundle';
```

Comment:
No `Path::join()` here?

Reply:
I did not use it anywhere in the tests for now.

Reply:
Ok.